src/main/java/de/tum/in/www1/artemis/domain/ProgrammingExercise.java

```diff
@@ -99,6 +99,10 @@
     @Transient
     private boolean isLocalSimulationTransient;
 
+    @Transient
+    @JsonProperty
+    private boolean checkoutSolutionRepository = false;
```

Comment:
Can you please document the property and its default value here? thanks!

Reply:
I've added documentation.